### PR TITLE
fix: fixes for codesandbox examples in docs

### PR DESCRIPTION
**Pull request by ccedrone**
From Accordion to Blockquote, fixed so far, ongoing

### CODE REVIEW

**Review by greenervigil — APPROVED** [review]

**Review by greenervigil — COMMENTED** [review]
so far so good

File: .changeset/codesandbox-fixes.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-docs": patch
```
Comment (silvalaura) [inline .changeset/codesandbox-fixes.md:2]: Can we make this major?

File: website/react-magma-docs/src/pages/api/form.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ The `Form` component is a relatively simple container. It is used to contain int
 
 ```typescript
 import React from 'react';
-import { Button, Input, Form, Spacer, SpacerAxis } from 'react-magma-dom';
+import { Button, ButtonType, Form, Input } from 'react-magma-dom';
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/api/form.mdx:16]: `Spacer, SpacerAxis` are still needed (used in line 27)

File: website/react-magma-docs/src/pages/api/icon-button.mdx [hunk 3]
```diff
@@ -344,7 +343,7 @@ export function Example() {
 
 Passing in the `testId` prop will add the `data-testid` attribute to the button element for easier querying in tests.
 
-```html
+```html noCodeSandbox
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/api/icon-button.mdx:346]: Should we make sure we consistently use `noCodeSandbox` for ALL Testing sections of the docs?

File: website/react-magma-docs/src/pages/api/theme.mdx [hunk 4]
```diff
@@ -30,20 +30,9 @@ import {
...
-    neutral400: '#BFBFBF',
-    neutral500: '#8f8f8f',
-    neutral600: '#707070',
-    neutral900: '#575757',
-    neutral800: '#3F3F3F',
-    neutral700: '#ffffff',
-    border: magma.colors.secondary600,
+    primary: `${magma.colors.tertiary500}`,
+    focus: `${magma.colors.focusInverse}`,
+    danger: `${magma.colors.danger500}`,
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/api/theme.mdx:35]: These changes should be reversed. @greenervigil recently updated this correctly in https://github.com/cengage/react-magma/pull/723/files#diff-c8fb829e210501c88af80b591bba017744a762c34bcee8b5e55c3628714b5f33R33-R46.

File: website/react-magma-docs/src/pages/api/tooltip.mdx [hunk 5]
```diff
@@ -207,20 +211,11 @@ export function Example() {
...
-    inverse: {
-      backgroundColor: magma.colors.neutral100,
-      textColor: magma.colors.neutral,
-    }
- }
+```html noCodeSandbox
+tooltip: { arrowSize: '6px', backgroundColor: magma.colors.neutral, fontWeight:
+600, maxWidth: '300px', textColor: magma.colors.neutral100, typeScale:
+magma.typeScale.size01, zIndex: 999, inverse: { backgroundColor:
+magma.colors.neutral100, textColor: magma.colors.neutral, } }
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/api/tooltip.mdx:218]: I think this looks better / easier to read with the previous formatting

**Review by silvalaura — APPROVED** [review]

**Pull request merged**